src/server/helpers/app.test.ts

```diff
@@ -55,6 +60,7 @@ vi.mock('../config', async (requireOriginal) => {
 describe('server/helpers/app', () => {
   const digidClientId = oidcConfigDigid.clientID;
   const eherkenningClientId = oidcConfigEherkenning.clientID;
+  let resMock = new ResponseMock() as unknown as Response;
```

Comment:
Kun je er voor zorgen dat ResponseMock automatisch die Response type krijgt? Dan hoef je dat niet elke keer te typen. Je kunt ook een functie-tje maken als:

```
getResponseMock => new ResponseMock() as unknown as Response
```

Reply:
Ah! heel nice! on it

Reply:
Heb er een functie aangehangen die het zo terug geeft als je zegt. Scheelt weer, dat `as unknown as Response` was wel meh